README.md

```diff
@@ -6,3 +6,7 @@ AppVeyor: [![AppVeyor](https://ci.appveyor.com/api/projects/status/github/aspnet
 ASP.NET Templates provide project templates which are used in .NET Core for creating ASP.NET Core applications.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+## Building Templates
+- Running build.cmd in this repo requires NPM which can be installed from https://nodejs.org/en/.
+- The ASP.NET localhost development certificate must also be installed and trusted or else you'll get a test error "Certificate error: Navigation blocked".
```

Comment:
How do I do this, or check that it's done?

Reply:
Ask @javiercn, he's still writing those instructions 😄

Reply:
The tests put the cert in place for you automatically, there are a couple of options:
* Setup ASPNETCORE_BROWSER_AUTOMATION_FIREFOX=true to run the automation tests with Firefox.
* Trust the certificate on your machine:
  * Manually copying the localhost certificate on the personal current user store to the trusted root certification authorities user store
  * Running the dev-certs tool on your machine (one time operation) to setup the certificate to trusted.
    * Clone the dotnet tools repo, build and run `dotnet <<path-to-dev-certs>>.dll https -t` (accept when prompted)
  * Install the tool as a global tool using `dotnet tools install dev-certs --nugetconfig <<nugetconfigfilewithourmygetfeed>>`
    * Run dotnet dev-certs https -t

Reply:
Nice. Put those instructions in the dev certs readme and I'll link to them from here.

Reply:
Are we just waiting for the dev certs docs? @javiercn , is that done, so that it can be linked from here?

Reply:
Yes. @javiercn declined to document the current process because he said the real process will be available in a few weeks.

Reply:
Ok